File: Salario.py
nombre = input("Ingrese su nombre: ")
salario = int(input("Ingrese su salario por hora: "))
horas = int(input("Ingrese sus horas trabajadas al mes: "))
salariomes = horas*salario

if (salariomes > 450000):
  print("Su nombre es",nombre.upper()," Y su salario mensual es: $",salariomes)
else:
  print(nombre)

File: Salario.cpp
#include <iostream>
using namespace std;

float horas,salario,salariomes;
string nombre;
int main(){
  cout<<"Ingrese su nombre: "<<endl;
  cin>>nombre;
    cout<<"Ingrese su salario por hora: "<<endl;
  cin>>salario;
    cout<<"Ingrese sus horas trabajadas al mes: "<<endl;
  cin>>horas;
  float salariomes = horas*salario;
if (salariomes > 450000){
  cout<<"Su nombre es: "<<nombre<<" Y su salario es de: $"<<salariomes<<endl;
}
  else{
  cout<<nombre<<endl;
}
}